WalletWasabi.Fluent/Styles/Styles.xaml

```diff
@@ -121,5 +125,127 @@
     <Setter Property="(TextBlock.FontWeight)" Value="Bold" />
     <Setter Property="Margin" Value="0 0 0 10" />
   </Style>
+
+  <!--Check mark-->
+  <Style Selector=":is(PathIcon).checkMark">
+    <Setter Property="Foreground" Value="{DynamicResource SystemControlBackgroundAccentBrush}" />
+    <Setter Property="Width" Value="15" />
+    <Setter Property="Height" Value="15" />
+    <Setter Property="Data" Value="{StaticResource checkmark_filled}" />
+    <Setter Property="Opacity" Value="0" />
+    <Setter Property="Transitions">
+      <Transitions>
+        <DoubleTransition Property="Opacity" Duration="0:0:0.3" />
+      </Transitions>
+    </Setter>
+  </Style>
+
+  <Style Selector="DataValidationErrors">
+    <Setter Property="Template">
+      <ControlTemplate>
+        <DockPanel LastChildFill="True">
+          <ContentControl
+            DockPanel.Dock="Bottom"
+            ContentTemplate="{TemplateBinding ErrorTemplate}"
+            DataContext="{TemplateBinding Owner}"
+            Content="{Binding (DataValidationErrors.Errors)}"
+            IsVisible="{Binding (DataValidationErrors.HasErrors)}" />
+          <ContentPresenter
+            Name="PART_ContentPresenter"
+            Background="{TemplateBinding Background}"
+            BorderThickness="{TemplateBinding BorderThickness}"
+            ContentTemplate="{TemplateBinding ContentTemplate}"
+            Content="{TemplateBinding Content}"
+            Padding="{TemplateBinding Padding}" />
+        </DockPanel>
+      </ControlTemplate>
+    </Setter>
+    <Setter Property="ErrorTemplate">
+      <DataTemplate>
+        <ItemsControl Items="{Binding}">
+          <ItemsControl.ItemTemplate>
+            <DataTemplate>
+              <TextBlock Text="{Binding Message}" Foreground="{Binding Severity, Converter={StaticResource ErrorSeverityColorConverter}}" TextWrapping="Wrap" />
+            </DataTemplate>
+          </ItemsControl.ItemTemplate>
+        </ItemsControl>
+      </DataTemplate>
+    </Setter>
+  </Style>
+
+  <Style Selector="TextBox">
```

Comment:
i wonder if we want to maintain this textbox template? i hope we can use specific styles instead of copying the textbox template wholesale again like we did in the past. @danwalmsley

Reply:
if we do want to include this, we should probably move this to it's own xaml file.

Reply:
Agree... we can move it in a sperate PR.